spog/api/src/config/default.yaml

```diff
@@ -5,6 +5,7 @@ $schema: "../../../model/schema/config.json"
 global:
   documentationUrl: https://github.com/trustification/trustification
   supportUrl: https://github.com/trustification/trustification/issues
+  expandedPageUrl: https://github.com/trustification/trustification/issues
```

Comment:
I am not sure I understand the name. It should be something like "support case", "issue", "ticket"?